tests/integration_tests/modules/test_apt.py

```diff
@@ -156,6 +156,8 @@ def test_ppa_source(self, class_client: IntegrationInstance):
         assert (
             "http://ppa.launchpad.net/simplestreams-dev/trunk/ubuntu"
             in ppa_path_contents
+            or "https://ppa.launchpadcontent.net/simplestreams-dev/trunk/ubuntu/"
```

Comment:
Looks like flake8 is unhappy due to line length. Maybe something that also is more specific to Jammy-only as far as conditionals?

That way our test is more prescriptive/strict on each series so we know exactly what
we expect and where we expect it.

```diff
diff --git a/tests/integration_tests/modules/test_apt.py b/tests/integration_tests/modules/test_apt.py
index 6b3a8b7c..46a5e301 100644
--- a/tests/integration_tests/modules/test_apt.py
+++ b/tests/integration_tests/modules/test_apt.py
@@ -153,8 +153,9 @@ class TestApt:
             "simplestreams-dev-ubuntu-trunk-{}.list".format(release)
         )
 
+        host = "launchpad" if release == "jammy" else "launchpadcontent"
         assert (
-            "http://ppa.launchpad.net/simplestreams-dev/trunk/ubuntu"
+            f"http://ppa.{host}.net/simplestreams-dev/trunk/ubuntu"
             in ppa_path_contents
         )

```

Reply:
Oops, I fixed format locally but forgot to push the commit. I'll include your suggestion in my next push.